tests/test_scripts.py

```diff
@@ -77,7 +77,8 @@ def test_expert_demos_rollouts_from_policy(tmpdir):
 
 
 EVAL_POLICY_CONFIGS = [
-    {},
+    {"videos": True},
+    {"videos": True, "video_kwargs": {"single_video": True}},
```

Comment:
Oh, looks like we should actually be using `single_video=False` to increase coverage, actually.
```suggestion
    {"videos": True, "video_kwargs": {"single_video": False}},
```